lib/pdn/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2010 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config PDN_LIBRARY
```

Comment:
Nearly everything is a library. Drop the _LIBRARY suffix.

Reply:
ok, thanks.